javaslang/src/main/java/javaslang/collection/Multimap.java

```diff
@@ -147,7 +147,7 @@
 
     @Override
     default boolean contains(Tuple2<K, V> element) {
-        return get(element._1).map(v -> Objects.equals(v, element._2)).getOrElse(false);
+        return get(element._1).map(v -> v.contains(element._2)).getOrElse(false);
```

Comment:
I think contains on `Multimap` was broken - it should return true if one of the values is associated with the key? It that right